### PR TITLE
Add in Bastion damage messages.

### PR DESCRIPTION
**Pull request by KaloHG**
Add in a basic listener to send messages when the bastion takes damage.

### CODE REVIEW

**Comment by Maxopoly** [comment]
Aren't there already messages for all of these except for elytra (which literally makes you explode)

**Comment by KaloHG** [comment]
Well, currently you merely get a massive spam of messages saying "Bastion prevented block placement." Which is not very intuitive. 
This adds in messages as requested by issue #23 .

**Comment by Maxopoly** [comment]
But now you get two messages for everything

File: src/main/java/isaac/bastion/listeners/BastionDamageListener.java [hunk 1]
```diff
@@ -127,6 +129,23 @@ public void onBucketEmpty(PlayerBucketEmptyEvent event) {
...
+		if (event.getCause() == BastionDamageEvent.Cause.BLOCK_PLACED) {
+			event.getPlayer().sendMessage(ChatColor.RED + "Block placement damaged Bastion.");
+		}
+		if (event.getCause() == BastionDamageEvent.Cause.PEARL) {
+			event.getPlayer().sendMessage(ChatColor.RED + "Pearling damaged Bastion.");
+		}
+		if (event.getCause() == BastionDamageEvent.Cause.ELYTRA) {
+			event.getPlayer().sendMessage(ChatColor.RED + "Elytra flight damaged Bastion.");
+		}
+		if (event.isCancelled() == true) {
```
Comment (Maxopoly) [inline src/main/java/isaac/bastion/listeners/BastionDamageListener.java]: Also never do logic like this, just `if (event.isCancelled())`

**Comment by KaloHG** [comment]
So, we could choose to remove the messages I added in except for the isCancelled() message. However since /config lets you disable the entire "Block placement..." message, I feel this is a reasonable addition.

**Comment by Maxopoly** [comment]
Listening for your own events is kind of a bad habit, especially when you are basically just switch casing over where the call came from. You should be adding proper messages in the places these events are called from instead.

**Comment by KaloHG** [comment]
closed cause dead PR.

**Pull request closed without merging**